### ISSUE TITLE
sorting in get_dictionary_values is a terrible approach

### ISSUE THREAD

**Issue by 1dziubo1** [robotframework/robotframework#3077]
My dict:
{'February 1, 2019': <MainDir.SubDir.Class object at 0x03CE4650>,
 'February 4, 2019': <MainDir.SubDir.Class object at 0x03C71490>,
 'January 22, 2019': <MainDir.SubDir.Class object at 0x03CD8B90>,
 'January 23, 2019': <MainDir.SubDir.Class object at 0x03C710F0>,
 'January 24, 2019': <MainDir.SubDir.Class object at 0x03CD8AF0>,
 'January 25, 2019': <MainDir.SubDir.Class object at 0x03CD8BF0>,
 'January 28, 2019': <MainDir.SubDir.Class object at 0x03CE4370>,
 'January 29, 2019': <MainDir.SubDir.Class object at 0x03CE4B70>,
 'January 30, 2019': <MainDir.SubDir.Class object at 0x03CE4670>,
 'January 31, 2019': <MainDir.SubDir.Class object at 0x03CE4730>}

1) This is NOT expected behavior. Not any user will expect, that getting values forces sorting!
2) Approach is anti=pattern, where function should do only one thing
3) In many cases sort will not work as intended (see the one above -yes, keys are strings, not date, this is intended). What will happen if dict will have object type as key?
4) User cannot use this keyword without sorting
(This can be solved by simple: Get Dictionary Values  ${dict}  sort=True)

**Comment by pekkaklarck** [comment]
I disagree that sorting is a bad idea or antipattern in this case, but would be fine making it configurable. We actually even have a comment about that in `Collections.get_dictionary_keys` that `get_dictionary_values` uses internally. Anyone interested to implement that? Ought to be easy and could be included already in RF 3.1.2.

**Comment by MaciejWiczk** [comment]
Hi @1dziubo1 , @pekkaklarck. Could You take a look at #3094 ?

**Comment by pekkaklarck** [comment]
One a very quick look #3094 looks pretty good. No time for real review but I add this issue to RF 3.1.2 scope so that it won't be forgotten.
